language/move-lang/README.md

```diff
@@ -17,15 +17,44 @@ Move source language is an ergonomic language for writing Modules and Scripts th
 Move source language is an expression-based language that aims to simplify writing Move programs---modules and scripts---without hiding the core concepts in Move bytecode.
 
 Currently, there are command line tools for Move.
+
 * Move Check is used for checking code, but it does not generate bytecode
 * Move Build is used for checking and then compiling to bytecode
+
 In the future there should be other utilities for testing and play grounding the Move modules.
 
 There is unfortunately no documentation for the language syntax or features. See the stdlib for examples.
 
+## Values
+
+### Mission
+
+Deliver a minimalistic, expressive, safe, and transparent language to produce--and link with--Move bytecode.
+
+### Primary Values
+
+* **More Concise as Bytecode** Move is expression based, which often allows for more concise and composed programs without the need for extra locals or structure. Additionally, expressions allow for programming directly on the stack in a controlled environment; however, sometimes the language allows "stack manipulation" in a way that the bytecode would not allow (due to it's stack balancing requirements), and additional locals will be generated by the compiler.
+
+* **Move Bytecode Transparency** The Move source language tries to lift up concepts in the Move bytecode into a source language; it is not trying to hide them. The bytecode already has some strong opinions (much stronger than you might expect to find in a bytecode language), the source language is trying to keep that programming model and line of thinking. This means direct access to references, global storage, stack values, etc. The intention of this value is to remove the need of ever needing to write bytecode directly. Additionally, this means full interoperability with functions and types declared in published modules.
+
+* **Stricter than Bytecode** The source language often adds additional level of restrictions. At an expression level, this means no arbitrary manipulation of the stack (only can do so through expressions), and no dead code or unused effects. At a module level this could mean additional warnings for unused types or un-invocable functions. And will also mean mean adding integration for formal verification.
```

Comment:
"additional level" needs either an article or to be made plural. The last sentence has a repeated word ("mean mean"), is missing a subject, and should not start with "And"; moreover, I'm not sure that integrating support for formal verification fits wells with "stricter than bytecode". You could say that it is related, but since we don't yet have anything to say on that topic, you could also leave it out.

Reply:
> I'm not sure that integrating support for formal verification fits wells with "stricter than bytecode". You could say that it is related, but since we don't yet have anything to say on that topic, you could also leave it out.

I tried to add a different qualification, but its a bit wordy. Let me know what you think

Reply:
Yes, it looks much better now.